Formula/m/malcontent.rb

```diff
@@ -0,0 +1,20 @@
+class Malcontent < Formula
+  desc "Supply Chain Attack Detection, via context differential analysis and YARA"
+  homepage "https://github.com/chainguard-dev/malcontent"
+  url "https://github.com/chainguard-dev/malcontent/archive/refs/tags/v1.6.0.tar.gz"
+  sha256 "a8538888caee8d16fce62c398fc8eaa1af9aca225139afe9c8247c84202403db"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+  depends_on "pkgconf" => :build
+  depends_on "yara"
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w", output: bin/"mal"), "./cmd/mal"
+  end
+
+  test do
+    (testpath/"test.py").write("subprocess. python')")
+    assert_match "execute external program", shell_output("#{bin}/mal analyze #{testpath}/test.py")
```

Comment:
sorry, I was not clear, you can still keep the version check, it is helpful for both ends :)